### DISCUSSION TITLE
TownyBuildEvent | Usage

### DISCUSSION THREAD

**Discussion by Fe1son** [TownyAdvanced/Towny#6652] [Q&A] [answered]
Hello, I have my own addon for towny. There is a problem, when using TownyBuildEvent, it is not possible to build if the chunk has an owner or the chunk is unowned (maybe). I need players to be able to build anyway. Perhaps I need to use a different event? Can I please have a code example?

**Comment by LlmDl** [comment, marked as answer]
You can use the TownyBuildEvent to override Towny's decision-making in any way you want. 

```java
@Eventhandler
public void onBuild(TownyBuildEvent event) {
    event.setCancelled(false);
}
```

Is enough to make a build event not cancelled. You just need to add some logic before you do that.
